hypervisor/arch/x86/mtrr.c

```diff
@@ -37,21 +37,17 @@ static struct fixed_range_mtrr_maps fixed_mtrr_map[FIXED_RANGE_MTRR_NUM] = {
 	{ MSR_IA32_MTRR_FIX4K_F8000, 0xF8000U, 0x1000U },
 };
 
-static bool is_fixed_range_mtrr(uint32_t msr)
-{
-	return (msr >= fixed_mtrr_map[0].msr)
-		&& (msr <= fixed_mtrr_map[FIXED_RANGE_MTRR_NUM - 1U].msr);
-}
-
-static uint32_t get_index_of_fixed_mtrr(uint32_t msr)
+static bool get_index_of_fixed_mtrr(uint32_t msr, int *index)
 {
```

Comment:
I prefer u return index here, but define an XXX_INVALID_INDEX...  The caller need to check if it is INVALID_INDEX

Reply:
Sure, will do